client/profile-wizard/steps/product-types/index.js

```diff
@@ -40,12 +40,22 @@ export class ProductTypes extends Component {
 			error: null,
 			isMonthlyPricing: true,
 			selected: profileItems.product_types || defaultProductTypes,
+			isWCPayInstalled: false,
 		};
 
 		this.onContinue = this.onContinue.bind( this );
 		this.onChange = this.onChange.bind( this );
 	}
 
+	componentDidMount() {
```

Comment:
This did test well, but I wonder if we could run into an instance where the `installedPlugins` isn't loaded yet, but this function is triggered?
You could also set the initial value in the state to `null`, and in the render method add something like this:
```
if ( this.state.isWCPayInstalled === null && this.props.installedPlugins ) {
this.setState( {
			isWCPayInstalled: installedPlugins.includes(
				'woocommerce-payments'
			),
		} );
}
```